src/app/[locale]/rest-client/page.tsx

```diff
@@ -27,115 +27,97 @@ export default function RestClientPage() {
     setError(null);
     setResponse(null);
 
+    if (values.contentType === ContentType.JSON && values.data?.trim()) {
+      const isJsonValid = validateJson(values.data, values.contentType);
```

Comment:
If you destructurize as indicated in the comment to the file

```suggestion
      const isJsonValid = validateJson(data, contentType);
```

Reply:
Thank you, I have done it like that!